tsl/src/continuous_aggs/materialize.c

```diff
@@ -1464,3 +1452,97 @@ continuous_aggs_completed_threshold_set(int32 materialization_id, int64 complete
 		relation_close(rel, NoLock);
 	}
 }
+static bool
+hypertable_get_min_and_max(SchemaAndName hypertable, Name time_column, int64 search_start,
+						   Oid time_type, int64 *min_out, int64 *max_out)
+{
+	Datum last_time_value;
+	Datum first_time_value;
+	bool val_is_null;
```

Comment:
Nit: it is usually good practice to initialize variables. In this case, `SPI_getbinval` is guaranteed to set the value, but it is a good measure to avoid bugs because of unpredictable values of the variable.